packages/clay-form/TOGGLE_SWITCH.mdx

```diff
@@ -11,23 +11,31 @@ import {RadioToggle, Toggle} from '$packages/clay-form/docs/toggle';
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
--   [Example](#example)
--   [With ClayRadioGroup](#usage-with-clayradiogroup)
+-   [Composing](#composing)
+    -   [Inside ClayRadioGroup](#inside-clayradiogroup)
 -   [API](#api)
 
 </div>
 </div>
 
-## Example
+## Composing
```

Comment:
Lets not use "composing" unless we are referencing our doc on it at https://clayui.com/docs/get-started/composing.html

Based on issue #3229, we can call either call this "Introduction", "Overview", or just leave it out. I think I would actually opt for just leaving no header for the first basic example that we showcase

Reply:
Oh, okay, I assumed the "condition" to use the `composing` heading was if we're describing using component props/subcomponents to compose a component